src/App.vue

```diff
@@ -1,8 +1,9 @@
 <template>
-  <div>
+  <div CLASS=" mx-auto max-w-[1400px] py-[140px] bg-info px-5">
```

Comment:
not sure if this matters but class is uppercased.

Reply:
fixed.